Sources/Plasma/Apps/plClient/CMakeLists.txt

```
@@ -1,3 +1,17 @@
+set(external_SCRIPTS
+    external/create_resource_dat.py
+    external/makeres.py
+    external/render_svg.py
+    external/scalergba.py
+)
+
+set(external_SCRIPTS
+    external/create_resource_dat.py
+    external/makeres.py
+    external/render_svg.py
+    external/scalergba.py
+)
+
```

```suggestion
set(external_SOURCES
    external/Cursor_Base.svg
    external/Linking_Book.svg
    external/Loading_Text_rasterfont.svg
    external/Voice_Chat.svg
)
```
I don't think we need it twice to make up for not having it at all.

---

:trollface: